nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamServiceImpl.java

```diff
@@ -155,6 +155,10 @@ public void start(ComponentContext context) {
     }
 
     protected void initProcessor(StreamProcessorDescriptor descriptor) {
+        if (! descriptor.isEnabled()) {
+            log.info("Processor {} disabled", descriptor.getId());
```

Comment:
Could use `descriptor::getId`

Reply:
yes, but other access are done like this and there is no performance problem here, so I prefer to be consistent